packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -3564,7 +3564,6 @@ export interface IDialogContentProps extends React.ClassAttributes<DialogContent
     draggableHeaderClassName?: string;
     isMultiline?: boolean;
     onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
-    // Warning: (ae-forgotten-export) The symbol "ResponsiveMode" needs to be exported by the entry point index.d.ts

```

Comment:
We should perhaps start breaking the build on warnings... this *was* called out. :)

Reply:
are there others?

Reply:
I am wondering if this is a new feature of api-extractor. First time I've seen this.

Reply:
IIRC the warning feature has existed but it is now capturing more symbols due to improvements in AE7. @natalieethell has more context.

Reply:
Yeah Kevin's right! The warnings were around before, they just looked a little different:

```
// WARNING: The type "ICalloutState" needs to be exported by the package (e.g. added to index.ts)
```

And like Kevin said, AE7 is also now capturing more symbols.

Reply:
Just checked and there are 38 of these warnings in office-ui-fabric-react.api.md